splice/commands.py

```diff
@@ -265,6 +266,59 @@ def ingest_tiles(in_file, channel_id, out_path, console_out, deploy_flag, verbos
         traceback.print_exc()
 
 
+@DataCommand.option("-v", "--verbose", action="store_true", dest="verbose", help="turns on verbose mode", default=False, required=False)
+@DataCommand.option("-q", "--quiet", action="store_true", dest="quiet", help="turns on quiet mode", default=False, required=False)
+@DataCommand.option("-l", "--leniency-minutes", type=int, dest="leniency", help="Leniency period in minutes for scheduling", default=15)
```

Comment:
I would like a few more comments on 'leniency'.  It is not obvious why you need it, and it makes my inner 'distributed systems guy' feel queesy.